webapp/.env.default

```diff
@@ -1,2 +1,3 @@
 BACKEND_API_PROTOCOL='https'
 BACKEND_API_DOMAIN='dev.verta.ai'
+BACKEND_API_BASE_PATH='/'
```

Comment:
If modeldb is being a reverse proxy at /my/modeldb then set this to `/my/modeldb/`